src/funcs.c

```diff
@@ -2825,3 +2825,71 @@ void joy_right()
   set(app->LV_GamesList, MUIA_List_Active, ind-1);
 
 }
+
+ULONG get_wb_version()
+{
+	static ULONG ver = 0UL;
+
+	if (ver != 0UL)
+	{
+		return ver;
+	}
+
+	if (WorkbenchBase == NULL)
```

Comment:
Where do you set WorkbenchBase?

Reply:
WorkbenchBase is an `extern` out of proto/wb.h, and is set automatically (probably by libauto).

Reply:
I am sure -lauto opens it. But should we slowly change the code and open them manually, especially when we add new libraries? I think this is a good practice.

Reply:
We were already making use of workbench.library before this branch, so it's not a matter of adding a new library. Rather this is something that should be looked at separately, when we do excise libauto from the project.